scrapy_playwright/handler.py

```diff
@@ -39,15 +40,50 @@
     _maybe_await,
 )
 
+# Supporting for Windows
+if sys.platform == "win32" and sys.version_info >= (3, 8):
+    import threading
+
+    class Var:
+        windows_loop = None
+        windows_thread = None
+
+    def windows_get_asyncio_event_loop():
```

Comment:
Please make these functions underscored, ideally only the download handler should be public.